pkg/scalers/artemis_scaler.go

```diff
@@ -39,10 +39,9 @@ type artemisMetadata struct {
 //revive:enable:var-naming
 
 type artemisMonitoring struct {
-	Request   string `json:"request"`
-	MsgCount  int    `json:"value"`
-	Status    int    `json:"status"`
-	Timestamp int64  `json:"timestamp"`
+	MsgCount  int   `json:"value"`
+	Status    int   `json:"status"`
+	Timestamp int64 `json:"timestamp"`
```

Comment:
Artemis scaler was failing to deserialize json response from artemis endpoint because it has type string, but when I try 

```
$ curl 'http://artemis:artemis@artemis-activemq.artemis:8161/console/jolokia/read/org.apache.activemq.artemis:broker="artemis-activemq",component=addresses,address="test",subcomponent=queues,routing-type="anycast",queue="test"/MessageCount'  | jq

{
  "request": {
    "mbean": "org.apache.activemq.artemis:address=\"test\",broker=\"artemis-activemq\",component=addresses,queue=\"test\",routing-type=\"anycast\",subcomponent=queues",
    "attribute": "MessageCount",
    "type": "read"
  },
  "value": 1000,
  "timestamp": 1607989283,
  "status": 200
}
```

I could update Request to be 

```go
Request struct {
		MBean     string `json:"mbean"`
		Attribute string `json:"attribute"`
		Type      string `json:"type"`
	} `json:"request"`
```

however it's not actually used anywhere, and since I know it used to work, I'm guessing some contract from Artemis changed? (not too sure) but since it's not used I figured removing it would be a safer change.